spec/61-span-id-format.md

```diff
@@ -0,0 +1,20 @@
+## Considerations for `span-id` field generation
+
+This section suggests some practices to consider when implementing
+`span-id` generation algorithms to ensure
+interoperability between different systems.
+
+### Uniqueness of `span-id`
+
+The value of `span-id` MUST be unique within a <a>distributed trace</a>.
+If the value of `span-id` is not unique within a distributed trace,
+parent-child relationships between spans within the distributed trace
+may be ambiguous.
+
+### Randomness of `span-id`
+
+Values of `span-id` SHOULD be randomly generated. Randomness of `span-id`
+addresses some [security](#security-considerations) and [privacy
+concerns](#privacy-considerations) of exposing unwanted information.
+Randomness also ensures a high probability, though not a guarantee, of
+uniqueness within a distributed trace.
```

Comment:
Majority of the concerns are related to the "trace-id" which identifies a "request". Not sure if this justify this.

Reply:
This wording is just to discourage people from using PII in span ids. It is more of a concern with trace ids, but I think it still applies to span ids.

Reply:
I'm ok with this because it says SHOULD, leaving the door open for different algorithm. In general, I think the spec should be less opinionated about how IDs are generated, because nunyabiznis, and should be just talking about the representation.